frontend/package.json

```
@@ -15,24 +18,24 @@
     "js-cookie": "^3.0.1",
     "mobx": "^6.5.0",
     "mobx-react-lite": "^3.4.0",
-    "@superwf/mobx-react-router": "^7.4.0",
     "moment": "^2.29.3",
+    "plural-ru": "^2.0.2",
     "react": "^18.0.0",
     "react-app-rewired": "^2.2.1",
+    "react-circular-progressbar": "^2.1.0",
+    "react-confirm-alert": "^3.0.6",
     "react-dom": "^18.0.0",
     "react-hook-form": "^7.29.0",
     "react-icons": "^4.3.1",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.0",
+    "react-textarea-autosize": "^8.3.4",
+    "react-textarea-with-suggest": "pazoozoo42/react-textarea-with-suggest#support-forwarded-ref",
```

```suggestion
    "react-textarea-with-suggest": pazoozoo42/react-textarea-with-suggest#7168b69,
```
Let's use the commit hash here, to ensure that unintended changes don't get accidentally deployed.

---

Actually, your PR got accepted (https://github.com/marylorian/react-textarea-with-suggest/pull/8), we can use the npm version.